updater/updater.go

```diff
@@ -44,6 +47,18 @@ var (
 	unsafeHTTP      = flag.Bool("unsafe", false, "Debug Only: Support non-https update checks for testing.")
 )
 
+const (
+	profileFileName          string = "updater-profile.conf"
+	keyIdentity              string = "identity"
+	keyChannel               string = "channel"
```

Comment:
we would probably no longer need keyIdentify and keyChannel constants.

Reply:
Oops :)
Removed